internal/cmd/db_restore.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/chiselstrike/iku-turso-cli/internal"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	dbCmd.AddCommand(restoreCmd)
+	restoreCmd.Flags().BoolVarP(&yesFlag, "yes", "y", false, "Confirms the restoration of all locations of the database.")
+	addDbTimestampFlag(restoreCmd)
+	restoreCmd.RegisterFlagCompletionFunc("instance", completeInstanceName)
+}
+
+var restoreCmd = &cobra.Command{
+	Use:               "restore database_name --timestamp 2023-09-25T14:53:00",
```

Comment:
I think timestamp should not be a flag but an argument. Without timestamp this operation makes no sense. You're also not checking if timestamp is set in the RunE. Which just proves my point.

Reply:
General idea was that in the future we might want to do restoration based on different kinds of parameters (sqld natively supports timestamp-based or generation-based restoration).